### PR TITLE
Ditto explorer UI: Improvements from review

### PR DESCRIPTION
**Pull request by thfries**
Dear ditto team,

this PR contains most of the changes proposed in the last PR. Next to that, the handling of different environments was improved, because that was very confusing.

Looking forward to your feedback. Thanks.

### CODE REVIEW

File: ui/modules/environments/environments.js [hunk 1]
```diff
@@ -60,65 +65,99 @@ export function ready() {
...
+function updateEnvSelector() {
+  let activeEnvironment = dom.environmentSelector.value;
+  if (!activeEnvironment || !environments[activeEnvironment]) {
+    activeEnvironment = Object.keys(environments)[0];
   };
-  for (const key of Object.keys(environments)) {
+
+  dom.environmentSelector.innerHTML = '';
+  Object.keys(environments).forEach((key) => {
+    // todo: replace with Utils.addDropDownEntries(dom.environmentSelector, [key]);
```
Comment (thjaeckle) [inline ui/modules/environments/environments.js:137]: What about this "todo"? Shall this be included in the PR?
We usually aim to not leave todos :)

**Review by thjaeckle — CHANGES_REQUESTED** [review]
Thx again @thfries for the work you put into the UI - great to see the improvements 👍 
I just found a "todo" left during a review - could you address that or do you want to do that at a later point?

File: ui/modules/environments/environments.js (hunk 1 above)
Comment (thfries) [reply ui/modules/environments/environments.js]: No, that is not critical at all. I just removed the todo. Good to know for the future.

**Review by thjaeckle — APPROVED** [review]
LGTM 👍 
If "ready" from your side, we could accept this PR and merge it.

**Comment by thjaeckle** [comment]
Thanks for the addition, merged the PR 👍

**Pull request merged**